package.json

```diff
@@ -27,7 +27,14 @@
     "require": [
       "babel-register",
       "babel-polyfill"
-    ]
+    ],
+    "babel": {
+      "presets": [
+        "es2015",
+        "stage-3",
+        "react"
+      ]
+    }
```

Comment:
Can you explain further what happens if you don't have this? 
Probably `babel-register` needs it and can't pick it up from the top level one. 
We should avoid the repetition, if you want I can take a look.